docs/how-to/h-deploy-azure.md

```diff
@@ -0,0 +1,242 @@
+> * Managed Identity and the Juju resources should be on the same Azure subscription
+> * The current setup has been tested on Ubuntu 22.04+
+
+### Juju 
+
+Install Juju via snap:
+
+```shell
+sudo snap install juju --channel 3.6/edge
+```
+
+Check that the Juju version is correctly installed:
+
+```shell
+juju version
+```
+
+which should show:
+
+```shell
+3.6-rc1-genericlinux-amd64
+```
+
+### Azure CLI
+
+Install the Azure CLI on Linux distributions by following the [Azure CLI's installation guide](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli-linux?pivots=apt).
+
+Verify that it is correctly installed:
+
+```shell
+az --version
+```
+
+The above command should produce output similar to the following:
+
+```shell
+azure-cli                         2.65.0
+
+core                              2.65.0
+telemetry                          1.1.0
+
+Dependencies:
+msal                              1.31.0
+azure-mgmt-resource               23.1.1
+
+Python location '/opt/az/bin/python3'
+Extensions directory '/home/deusebio/.azure/cliextensions'
+
+Python (Linux) 3.11.8 (main, Sep 25 2024, 11:33:44) [GCC 11.4.0]
+
+Legal docs and information: aka.ms/AzureCliLegal
+
+
+Your CLI is up-to-date.
+```
+
+### Authenticate
+
+> For more information on how to authenticate, refer to the [Juju documentation](/t/the-microsoft-azure-cloud-and-juju/1086) and [a dedicated user guide on how to register Azure on Juju](/t/how-to-use-juju-with-microsoft-azure/15219).
+
+First of all, retrieve your subscription-id
+
+```shell
+az login
+```
+
+After providing the authentication via web browser, you will be shown a list of information and a table with the subscriptions connected to your user, e.g.:
+
+```shell
+No     Subscription name                     Subscription ID                       Tenant
+-----  ------------------------------------  ------------------------------------  -------------
+[1] *  <subscription_name>                   <subscription_id>                     canonical.com
+[2]    <other_subscription_name>             <other_subscription_id>               canonical.com
+```
+
+In the prompt, select the subscription id you would like to connect the controller to, and store the id 
+as it will be needed in the next step when bootstrapping the controller. 
+
+### Bootstrap Juju controller on Azure
+
+First, you need to add a set of credentials to your Juju client:
+
+```shell
+juju add-credentials azure
+```
+
+This will start a script that will help you set up the credentials, where you will be asked:
+
+* `credential-name` — a sensible name that will help you identify the credential set, say `<CREDENTIAL_NAME>`
+* `region` — a default region that is most convenient to deploy your controller and applications. Note that credentials are not region-specific
+* `auth type` — authentication type. Select `interactive`, which is the recommended way to authenticate to Azure using Juju
```

Comment:
that authentication scheme seems unlikely to be useful??

Reply:
I took this form the [juju docs](https://juju.is/docs/juju/microsoft-azure)